packages/now-next/src/index.ts

```diff
@@ -744,13 +738,10 @@ export const build = async ({
           throw new Error('invariant: htmlFsRef != null && jsonFsRef != null');
         }
 
-        const outputPathPageHtml = outputPathPage.concat('.html');
-        prerenders[outputPathPageHtml] = htmlFsRef;
+        prerenders[outputPathPage] = htmlFsRef;
         prerenders[outputPathData] = jsonFsRef;
-        exportedPageRoutes.push({
-          src: path.posix.join('/', outputPathPage),
-          dest: outputPathPageHtml,
-        });
+        // we need to set the contentType since we removed the extension
+        (prerenders[outputPathPage] as FileFsRef).contentType = htmlContentType
```

Comment:
This cast to `FileFsRef` seems wrong because it could be one of two types `Prerender | FileFsRef`.

You should only assign the contenType of the FileFsRef objects or perhaps the Prerender object should also add a contenType property?

Reply:
Probably just do `htmlFsRef.contentType = htmlContentType` instead.

Reply:
Updated